### PR TITLE
`postcss-preset-env` : add `color-mix`

### PR DESCRIPTION
**Pull request by romainmenke**

### CODE REVIEW

File: plugin-packs/postcss-preset-env/package.json [hunk 1]
```diff
@@ -43,58 +43,59 @@
...
-		"@csstools/postcss-logical-viewport-units": "^1.0.0",
-		"@csstools/postcss-media-queries-aspect-ratio-number-values": "^1.0.0",
-		"@csstools/postcss-nested-calc": "^2.0.0",
-		"@csstools/postcss-normalize-display-values": "^2.0.0",
+		"@csstools/postcss-cascade-layers": "^3.0.1",
+		"@csstools/postcss-color-function": "^2.1.0",
+		"@csstools/postcss-color-mix-function": "^0.0.0",
+		"@csstools/postcss-font-format-keywords": "^2.0.2",
+		"@csstools/postcss-hwb-function": "^2.1.1",
+		"@csstools/postcss-ic-unit": "^2.0.2",
```
[the remark this answers is not in the document]
Comment (Antonio-Laguna) [reply plugin-packs/postcss-preset-env/package.json:51]: I think it doesn't hurt and people often come back checking versions from what we've seen

File: plugin-packs/postcss-preset-env/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ### Unreleased (unknown)
```
Comment (Antonio-Laguna) [inline plugin-packs/postcss-preset-env/CHANGELOG.md:3]: Should this be changed to minor? Not sure how the release will work through this

**Review by Antonio-Laguna — APPROVED** [review]
LGTM! Just a minor comment

File: plugin-packs/postcss-preset-env/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (romainmenke) [reply plugin-packs/postcss-preset-env/package.json:51]: True, that is indeed very useful :)

File: plugin-packs/postcss-preset-env/CHANGELOG.md (hunk 2 above)
Comment (romainmenke) [reply plugin-packs/postcss-preset-env/CHANGELOG.md]: Yes, I've changed this

**Pull request merged**